depthai.py

```diff
@@ -437,6 +513,10 @@ def show_landmarks_recognition(entries_prev, frame):
 
 stream_names = [stream if isinstance(stream, str) else stream['name'] for stream in config['streams']]
 
+enable_object_tracker = False
```

Comment:
Can be simplified a bit:

`enable_object_tracker = True if 'object_tracker' in stream_names else False`

Not necessarily better, just more compact. Just a suggestion.

Reply:
even simpler - `enable_object_tracker = 'object_tracker' in stream_names`